daemons/gptp/common/ptp_message.cpp

```diff
@@ -1354,56 +1352,37 @@ void PTPMessagePathDelayRespFollowUp::processMessage(IEEE1588Port * port)
 		/* Probably shouldn't happen either */
 		XPTPD_ERROR
 		    (">>> Received PDelay followup but no RESPONSE exists");
+
 		goto abort;
 	}
 
 	req->getPortIdentity(&req_id);
 	resp->getRequestingPortIdentity(&resp_id);
 
-	/*We need to keep track of lost respFUPs. If we have more
-	 * than a certain number of lost FUPs we should set asCapable
-	 * to false
-	 */
-	if( sequenceId != (port->getLastSeqId() + 1) )
-	{
-		XPTPD_ERROR("Current seqID: %d. %d are missing since the last one sent",
-				sequenceId, sequenceId-port->getLastSeqId());
-		if( (sequenceId - port->getLastSeqId() ) > port->getLostPdelayRespThresh() )
-		{
-			port->setAsCapable( false );
-		}
+	if( req->getSequenceId() != sequenceId ) {
+		XPTPD_ERROR
+			(">>> Received PDelay FUP has different seqID than the PDleay request (%d/%d)",
```

Comment:
This is just a silly typo, "PDleay" instead of "PDelay", but this would be a great time to fix it.